doc/nrf/test_and_optimize/test_framework/testing_unity_cmock.rst

```diff
@@ -32,20 +33,17 @@ To specify the APIs that should be mocked in a given test, edit :file:`CMakeList
 The relative path is needed if a file is included as, for example, :code:`#include <zephyr/bluetooth/gatt.h>`.
 In this case, the relative path is ``bluetooth``.
```

Comment:
i think that this path shall be updated to `zephyr/bluetooth`. Same in the code snippet:

mock_handle(${ZEPHYR_BASE}/include/zephyr/bluetooth/gatt.h `zephyr/bluetooth`)

That is because zephyr added one more directory in the includes `zephyr/include` -> `zephyr/include/zephyr`